### ISSUE TITLE
Accordion & Dropdown Do Not Work using 2.1.7

### ISSUE THREAD

**Issue by i2insights** [Semantic-Org/Semantic-UI#3649]
The Accordion and Dropdown elements do not appear to work.   I hope its a noob problem on my end.

I created a simple test page in Brackets to replicate Semantic UI elements (e.g., lists, labels, buttons, flags, columns, dividers, etc.).  

The accordion and dropdown elements render but are not functional.  i have one Javascript error that refers to line 497 of `semantic.js`.  The error reads `Cant't find variable: jQuery`  

Line 497 = 
`})( jQuery, window, document );`

My resources are:
`<script type="text/javascript" src="/semantic.js"></script>
 <link rel="stylesheet" href="/semantic.min.css">
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>`

I am using version 2.1.7.

Thank you in advance for your assistance.

**Comment by avalanche1** [comment]
Closing because no test case.
